inheritance/lib/src/test/java/inheritance/RestaurantTests.java

```diff
@@ -67,15 +67,4 @@ void testAddReviewUpdatesPriceCategory() {
     assertEquals("$$$", restaurant.priceCategory);
   }
 
-//  @Test
-//  void testRestaurantManagerAddReview() {
-//    RestaurantManager restaurantManager = new RestaurantManager();
-//    Review review = new Review("Sonic", 4, "$$", "Great food!", "Alice");
```

Comment:
Remember that your Review constructor accepts a Business object as its first parameter. For testing, you'll want to create a test business that you can pass into this constructor to create the new review. It will also require you to provide an `Optional.of("")`, where `""` is the title of the movie. If you don't want to do that for every review, you can overload the Review constructor by making two versions - one with the Optional and one without.